examples/example-event-handler/dist/ExampleSubgraph/abis/ExampleContract.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "anonymous": false,
+    "inputs": [{ "indexed": true, "name": "exampleParam", "type": "string" }],
+    "name": "ExampleEvent",
+    "type": "event"
+  }
+]
```

Comment:
I don't think we want to include the ./dist/ folder in the repo.  Might be best to add ./dist to .gitignore.

Reply:
I think it's good, so when reviewing we can see the impact of a change on the example.

Reply:
I can live with either approach.